docs/instant.md

```diff
@@ -562,11 +562,20 @@ The combination of `roundingIncrement` and `smallestUnit` must make an increment
 
 The `roundingMode` option controls how the rounding is performed.
 
-- `ceil`: Always round up, towards the end of time.
-- `floor`, `trunc`: Always round down, towards the beginning of time.
-  (These two modes behave the same, but are both included for consistency with `Temporal.Duration.round()`, where they are not the same.)
+- `ceil`: Always round up, towards the end of time (positive infinity).
+- `expand`:  Always round away from from zero like `ceil` for positive value and like `floor` for negative value.
+- `floor`: Always round down, towards the beginning of time (negative infinity).
+- `trunc`: Always round towards zero.
```

Comment:
"floor" and "trunc" are still the same thing for times, which are always treated as positive. I preferred the previous explanation. Similarly, I'd prefer to explain that `halfFloor` and `halfTrunc` are equivalent for times.